### ISSUE TITLE
Don't randomize voting options

### ISSUE THREAD

**Issue by marieknox** [jonhull/hd-vote#2]
When I put in the names for the BoD Election, I intentionally entered them in alphabetically by last name. Then it changed that :/

**Comment by jonhull** [comment]
Yes, it randomizes it for every person differently, I believe, to avoid any bias.  Glad to hear you are having an election again!

**Comment by marieknox** [comment]
That actually makes sense, avoiding bias. Maybe I was just annoyed because
I was trying to be too particular!

Me too. The old Board was... something. Having a rep of the people should
be a nice change of pace.
